Neos.Neos/Migrations/Mysql/Version20210126083531.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Neos\Flow\Persistence\Doctrine\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20210126083531 extends AbstractMigration
+{
+    public function getDescription() : string
+    {
+        return 'add index to dimensions hash';
+    }
+
+    /**
+     * @param Schema $schema
+     * @throws \Doctrine\DBAL\Exception
+     */
+    public function up(Schema $schema) : void
+    {
+        $this->abortIf($this->connection->getDatabasePlatform()->getName() !== 'mysql', 'Migration can only be executed safely on \'mysql\'.');
```

Comment:
Is this for a reason in a second migration? If not, it would be better to merge it with the previous one.

Reply:
I merged the two migrations.